host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -104,7 +106,7 @@
      * @param port          the port number.
      * @param securePort    the secure port number.
      */
-    @LogMessage(level = Level.INFO)
+    @LogMessage(level = INFO)
```

Comment:
Oops; this kind of cosmetic change wasn't intentional. I can reverse it if wanted.